source/code/plugin/in_kube_nodes.rb

```diff
@@ -191,6 +193,20 @@ def parse_and_emit_records(nodeInventory, batchTime = Time.utc.iso8601)
             capacityInfo = items["status"]["capacity"]
             ApplicationInsightsUtility.sendMetricTelemetry("NodeMemory", capacityInfo["memory"], properties)
 
+            begin
+              if (!capacityInfo["nvidia.com/gpu"].nil?) && (!capacityInfo["nvidia.com/gpu"].empty?)
+                properties["nvigpus"] = capacityInfo["nvidia.com/gpu"]
+              end
+
+              if (!capacityInfo["amd.com/gpu"].nil?) && (!capacityInfo["amd.com/gpu"].empty?)
```

Comment:
Is it a supported scenario to have both nvidia & AMD gpu in a single VM? if not its better to move the amd check as else if ?

Reply:
This is specifically meant for hybrids. EKS,GKE & AKS dont have mixed gpu SKUs (yet), but hybrids can have both in the same node, as they are just accelerator cards that goes into the ePCI slots.